### DISCUSSION TITLE
淘宝小程序的开发可以用吗，使用的时候有什么要注意的？

### DISCUSSION THREAD

**Discussion by tc9011** [ant-design/ant-design-mini#533] [Q&A]
Hi, 淘宝小程序可以用吗, https://github.com/ant-mini-program/mini-antui 之前的 lib 停止维护了，这个库对淘宝小程序的支持程度怎么样，谢谢

**Comment by Alpsli** [comment]
可以用的，对基础库 2.x 的无兼容性问题
